common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -271,7 +272,6 @@ private DefaultResourceLeak track0(T obj) {
 
     private void clearRefQueue() {
         for (;;) {
-            @SuppressWarnings("unchecked")
```

Comment:
Why useless? refQueue takes object.

Reply:
![image](https://user-images.githubusercontent.com/12047600/70198386-b67e3f80-1749-11ea-9694-f6fc49ffd5c7.png)

This was identified as unwanted in Intellij idea.

Reply:
There is a question in [stackoverflow](https://stackoverflow.com/questions/1129795/what-is-suppresswarnings-unchecked-in-java), this check is useful for generic type. This situation just a class cast... I guess that is the reason the unchecked marked as useless.